src/schema/re_frame.cljs

```diff
@@ -5,6 +5,16 @@
 (def ^:private ?cofx
   [:map])
 
+(def ^:private ?event-fx
+  [:maybe
```

Comment:
Do we really need a schema for the rf event? 🤔
Cc @shivekkhurana

Reply:
okay, imo it's a bit strange to have a proposal hidden within a pr labelled - "[Clean up biometrics and standard-auth and add tests](https://github.com/status-im/status-mobile/pull/18756/files#top)"
I think best to open up a separate pr completely for this purpose so it is easier to see the effects and not mix what we are trying to review here 👍

Reply:
good point. Will remove it from here.